app/javascript/components/zone-form/index.jsx

```diff
@@ -0,0 +1,59 @@
+import React, { useEffect, useState } from 'react';
+
+import MiqFormRenderer from '@@ddf';
+import createSchema from './zone-form.schema';
+import miqRedirectBack from '../../helpers/miq-redirect-back';
+import { API } from '../../http_api';
+
+const ZoneForm = ({ recordId }) => {
+  const [{ initialValues, isLoading }, setState] = useState({
+    isLoading: !!recordId,
+  });
+
+  const cancelUrl = recordId ? `/ops/zone_edit/${recordId}?button=cancel` : '/ops/zone_edit/?button=cancel';
+  const submitLabel = !!recordId ? __('Save') : __('Add');
+
+  const onSubmit = ({ href, id, created_on, updated_on, actions, authentications, ...data }) => {
```

Comment:
Hopefully if [this](https://github.com/ManageIQ/manageiq-ui-classic/pull/7431) gets merged, you would not need to do any filtering of fields here...